bin/mail/2016-05-02-petition-followup/send-test-mails

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+../send-to-mailing-list 5-2-petition-follow-up ../mailing-common/test-params-us.csv
```

Comment:
Q: Is the purpose of this to test the template and make sure the email is populated correctly from the information in the CSV?